network/Network.cc

```diff
@@ -1591,22 +1585,12 @@ Network::drivers(const Pin *pin)
     return nullptr;
 }
 
-void
-Network::clearNetDrvrPinMap()
-{
-  net_drvr_pin_map_.deleteContentsClear();
-}
-
 PinSet *
 Network::drivers(const Net *net)
 {
-  PinSet *drvrs = net_drvr_pin_map_.findKey(net);
```

Comment:
This change may no longer be needed since the dcalc code was updated to avoid the issue.

Reply:
Yes, dropped it.